### PR TITLE
changed prerequisites (added the setup of docker in different O.S)

### PR DESCRIPTION
**Pull request by srinivaspavan9**
This resolves #153

### CODE REVIEW

File: pages/prerequisites.mdx [hunk 1]
```diff
@@ -20,7 +20,8 @@ name: Prerequisites
    ```
    > We only use Yarn as our official Node package manager, and so we request you to use `Yarn` instead of `npm` and commit `yarn.lock` file.
 1. [Docker](https://docs.docker.com/get-docker/)
-
+	> For `Windows` users download and install `Docker` and `Docker ToolBox`.For `Mac` users `Docker` is enough.
```
Comment (pbteja1998) [inline pages/prerequisites.mdx:23]: For Mac, docker desktop needs to be downloaded. Also add links to where to download them.

Comment (srinivaspavan9) [reply pages/prerequisites.mdx]: Done

**Comment by duttakapil** [comment]
Is this PR ready to be merged?

**Comment by srinivaspavan9** [comment]
I have made the changes in the prerequisites.mdx which will help the collaborators for easy setup.

**Comment by pbteja1998** [comment]
I tried to setup docker in Ubuntu yesterday. I don't see why we need special instructions for installing docker in Linux. Anyone can just use the link I added (  https://docs.docker.com/get-docker/ ) for installing docker in their os. The only problem I faced is, I had to install docker compose separately. One can do that by using instructions from official docs itself( https://docs.docker.com/compose/install/ ).

**Comment by pbteja1998** [comment]
And why do you have two prs open? Close the one that is no longer relevant.

**Comment by pbteja1998** [comment]
Closing this since there is no response for a while. Please reopen it if necessary.

**Pull request closed without merging**